# HG changeset patch
# User Rik <rik@octave.org>
# Date 1546539583 28800
#      Thu Jan 03 10:19:43 2019 -0800
# Branch stable
# Node ID 6c65dafb72ca0677cf4c0cfdccf0596635062825
# Parent  bb8d66018bd7650002afb88e49abe258f56137d4
pt-jit.cc: Fix static analyzer detected issues (bug #55347).

* pt-jit.cc (Fjit_startcnt): Set name of function in warn_disabled_feature to
"jit_startcnt".

diff --git a/libinterp/parse-tree/pt-jit.cc b/libinterp/parse-tree/pt-jit.cc
--- a/libinterp/parse-tree/pt-jit.cc
+++ b/libinterp/parse-tree/pt-jit.cc
@@ -2924,12 +2924,12 @@ The original variable value is restored 
 @end deftypefn */)
 {
 #if defined (HAVE_LLVM)
   return SET_INTERNAL_VARIABLE_WITH_LIMITS (jit_startcnt, 1,
                                             std::numeric_limits<int>::max ());
 #else
   octave_unused_parameter (args);
   octave_unused_parameter (nargout);
-  warn_disabled_feature ("jit_enable", "JIT");
+  warn_disabled_feature ("jit_startcnt", "JIT");
   return ovl ();
 #endif
 }
